src/aiida/cmdline/commands/cmd_storage.py

```diff
@@ -164,3 +165,51 @@ def storage_maintain(ctx, full, no_repack, force, dry_run, compress):
     except LockingProfileError as exception:
         echo.echo_critical(str(exception))
     echo.echo_success('Requested maintenance procedures finished.')
+
+
+@verdi_storage.command('backup')
+@click.argument('dest', type=click.Path(file_okay=False), nargs=1)
+@click.option(
+    '--keep',
+    default=1,
+    help='Number of previous backups to keep in the destination. (default: 1)',
+)
+@click.option(
+    '--pg_dump_exe', type=click.STRING, default='pg_dump', help="Specify the 'pg_dump' executable, if not in PATH."
+)
+@click.option(
+    '--rsync_exe',
+    type=click.STRING,
+    default='rsync',
+    help="Specify the 'rsync' executable, if not in PATH. Used for both local and remote destinations",
+)
```

Comment:
Just stating again that I don't really like having these options here. First, they are backend specific, and second I don't really see why this should be the responsibility of the CLI/API and the user should not simply make sure these executables are in the path.

Reply:
We can maybe rediscuss this in the next meeting? I agree that if the backend is doesn't use postgresql, it would be strange to have the `pg_dump_exe` as an argument. But your proposition is just to say that "make sure rsync and other executables are in PATH"? But somehow we probably should list what executables are needed for which backend.

Reply:
@sphuber we discussed this a bit and thought to keep this for now, but we can also discuss further, if you feel strongly about this. I just currently added to the CLI that the `pg-dump-exe` argument is "Only needed for a PostgreSQL-based backend".